pkg/eventprocessor/impl/executor/impl/txnscope_createtable.go

```diff
@@ -0,0 +1,81 @@
+package impl
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/textileio/go-tableland/internal/tableland"
+	"github.com/textileio/go-tableland/pkg/parsing"
+	"github.com/textileio/go-tableland/pkg/tables/impl/ethereum"
+)
+
+func (ts *txnScope) executeCreateTableEvent(
+	ctx context.Context,
+	e *ethereum.ContractCreateTable,
+) (eventExecutionResult, error) {
```

Comment:
You'll note that each method that has the logic to process an event type returns a `eventExecutionResult` (see [here](https://github.com/tablelandnetwork/go-tableland/blob/750cb2793dfaaefe42d119d9c7aa1a7c04ed5631/pkg/eventprocessor/impl/executor/impl/txnscope.go#L43)). 

This is even a more constrained struct compared to `TxnExecutionResult`. It only has the `TableID` and the potential `Error`. This is necessary since we can't exactly create a `TxnExecutionResult` in this layer, since we don't know which is the index of the event in the transaction. We're only interested in executing an event here. No need to pollute with more arguments that are only useful to build some high-level struct.